from utils.nesting_exception import NestingException

class EventSourceException(NestingException):
    pass
